Source/Particles/MultiParticleContainer.H

```diff
@@ -390,7 +390,7 @@ protected:
 
         MFItInfoCheckTiling(pc_src, pc_dsts...);
 
-        if (WarpXParticleContainer::do_tiling && amrex::Gpu::notInLaunchRegion()) {
+        if (WarpXParticleContainer::do_tiling) {
```

Comment:
When using GPUs, is `WarpXParticleContainer::do_tiling` always off?

Reply:
Yes: https://github.com/ECP-WarpX/WarpX/blob/development/Source/Initialization/WarpXAMReXInit.cpp#L74
However, it is not impossible that this could change in the future. 
Would you recommend keeping `&& amrex::Gpu::notInLaunchRegion()` here?

Reply:
Yes. Either keep `&& amrex::Gpu::notInLaunchRegion()` or add `#ifndef AMREX_USE_GPU` around that part. 